src/Neo/SmartContract/Native/LedgerContract.cs

```diff
@@ -76,6 +76,9 @@ internal override ContractTask PostPersistAsync(ApplicationEngine engine)
 
         internal bool Initialized(DataCache snapshot)
         {
+            if (snapshot is null)
```

Comment:
It's better to move when we will the argument, in this way it will be done for all syscalls